wenet/bin/train.py

```diff
@@ -164,6 +164,7 @@
     # Init asr model from configs
     model = init_asr_model(configs)
     print(model)
+    print('the number of model params: %s' % sum(p.numel() for p in model.parameters()))
```

Comment:
print('the number of model params: {}'.format(sum(p.numel() for p in model.prameters())), string with format is preferred in wenet